src/components/dashboard/dashboardComponent.tsx

```diff
@@ -4,6 +4,8 @@ import useAxiosEffect from '../generic/useAxiosEffect';
 import { dashboardService } from '../../services/dashboardService';
 import { prepareChartData } from '../charts/prepareChartData';
 import { ComponentStateHandler, getAggregatedState } from '../generic/componentStateHandler';
+import LiveActivityComponent from './liveActivityComponent';
+import { FaTrophy } from 'react-icons/fa';
 
 const DashboardComponent = () => {
     const dashboardState = useAxiosEffect<any>(
```

Comment:
Please use proper type instead of `any`

Reply:
It might be necessary to adjust `getAggregatedState` function to be something like this:
```
export const getAggregatedState = <T extends any[]>(
    ...funcs: ((this: AxiosRequestConfig | undefined) => Promise<any>)[]
): ((o: AxiosRequestConfig | undefined) => Promise<T>) => {
    return (options?: AxiosRequestConfig) => {
        return Promise.all(funcs.map((f) => f.apply(options))) as Promise<T>;
    };
};
```
and then you can do:
`useAxiosEffect<[Rating[], LiveActivity[]]>`